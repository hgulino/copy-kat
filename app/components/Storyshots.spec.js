import initStoryshots, {
  multiSnapshotWithOptions
} from '@storybook/addon-storyshots';
import { createSerializer } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme, { mount } from 'enzyme';

Enzyme.configure({ adapter: new Adapter() });

initStoryshots({
  test: multiSnapshotWithOptions({
    renderer: mount,
    snapshotSerializers: [createSerializer()]
  })
});
