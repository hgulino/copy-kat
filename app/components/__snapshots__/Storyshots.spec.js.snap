// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots ActionButton color 1`] = `
<div>
  <div
    className="makeStyles-root-32"
  >
    <div
      className="makeStyles-wrapper-35"
    >
      <button
        className="MuiButtonBase-root MuiButton-root makeStyles-button-34 makeStyles-button-38 MuiButton-contained"
        disabled={false}
        onBlur={[Function]}
        onClick={null}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type={null}
      >
        <span
          className="MuiButton-label"
        >
          <span
            className="makeStyles-title-33 makeStyles-title-37"
          >
            New Template
          </span>
        </span>
        <span
          className="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
  <div
    className="makeStyles-root-32"
  >
    <div
      className="makeStyles-wrapper-35"
    >
      <button
        className="MuiButtonBase-root MuiButton-root makeStyles-button-34 makeStyles-button-71 MuiButton-contained"
        disabled={false}
        onBlur={[Function]}
        onClick={null}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type={null}
      >
        <span
          className="MuiButton-label"
        >
          <span
            className="makeStyles-title-33 makeStyles-title-70"
          >
            New Template
          </span>
        </span>
        <span
          className="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots ActionButton disabled 1`] = `
<div>
  <div
    className="makeStyles-root-203"
  >
    <div
      className="makeStyles-wrapper-206"
    >
      <button
        className="MuiButtonBase-root MuiButton-root makeStyles-button-205 makeStyles-button-209 MuiButton-contained Mui-disabled Mui-disabled"
        disabled={true}
        onBlur={[Function]}
        onClick={null}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={-1}
        type={null}
      >
        <span
          className="MuiButton-label"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
          <span
            className="makeStyles-title-204 makeStyles-title-208"
          >
            New Template
          </span>
        </span>
      </button>
    </div>
  </div>
  <div
    className="makeStyles-root-203"
  >
    <div
      className="makeStyles-wrapper-206"
    >
      <button
        className="MuiButtonBase-root MuiButton-root makeStyles-button-205 makeStyles-button-241 MuiButton-contained Mui-disabled Mui-disabled"
        disabled={true}
        onBlur={[Function]}
        onClick={null}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={-1}
        type={null}
      >
        <span
          className="MuiButton-label"
        >
          <span
            className="makeStyles-title-204 makeStyles-title-240"
          >
            New Template
          </span>
        </span>
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots ActionButton icons 1`] = `
<div>
  <div
    className="makeStyles-root-113"
  >
    <div
      className="makeStyles-wrapper-116"
    >
      <button
        className="MuiButtonBase-root MuiButton-root makeStyles-button-115 makeStyles-button-119 MuiButton-contained"
        disabled={false}
        onBlur={[Function]}
        onClick={null}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type={null}
      >
        <span
          className="MuiButton-label"
        >
          <span
            className="makeStyles-title-114 makeStyles-title-118"
          >
            New Template
          </span>
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
  <div
    className="makeStyles-root-113"
  >
    <div
      className="makeStyles-wrapper-116"
    >
      <button
        className="MuiButtonBase-root MuiButton-root makeStyles-button-115 makeStyles-button-161 MuiButton-contained"
        disabled={false}
        onBlur={[Function]}
        onClick={null}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type={null}
      >
        <span
          className="MuiButton-label"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M3.9 12c0-1.71 1.39-3.1 3.1-3.1h4V7H7c-2.76 0-5 2.24-5 5s2.24 5 5 5h4v-1.9H7c-1.71 0-3.1-1.39-3.1-3.1zM8 13h8v-2H8v2zm9-6h-4v1.9h4c1.71 0 3.1 1.39 3.1 3.1s-1.39 3.1-3.1 3.1h-4V17h4c2.76 0 5-2.24 5-5s-2.24-5-5-5z"
            />
          </svg>
          <span
            className="makeStyles-title-114 makeStyles-title-160"
          >
            New Template
          </span>
        </span>
        <span
          className="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots ActionButton loading 1`] = `
<div>
  <div
    className="makeStyles-root-295"
  >
    <div
      className="makeStyles-wrapper-298"
    >
      <button
        className="MuiButtonBase-root MuiButton-root makeStyles-button-297 makeStyles-button-301 MuiButton-contained Mui-disabled Mui-disabled"
        disabled={true}
        onBlur={[Function]}
        onClick={null}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={-1}
        type={null}
      >
        <span
          className="MuiButton-label"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
          <span
            className="makeStyles-title-296 makeStyles-title-300"
          >
            New Template
          </span>
        </span>
      </button>
      <div
        className="MuiCircularProgress-root makeStyles-buttonProgress-299 makeStyles-buttonProgress-302 MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
        role="progressbar"
        style={
          Object {
            "height": 24,
            "width": 24,
          }
        }
      >
        <svg
          className="MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
            cx={44}
            cy={44}
            fill="none"
            r={19.5}
            strokeWidth={5}
            style={Object {}}
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    className="makeStyles-root-295"
  >
    <div
      className="makeStyles-wrapper-298"
    >
      <button
        className="MuiButtonBase-root MuiButton-root makeStyles-button-297 makeStyles-button-345 MuiButton-contained Mui-disabled Mui-disabled"
        disabled={true}
        onBlur={[Function]}
        onClick={null}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={-1}
        type={null}
      >
        <span
          className="MuiButton-label"
        >
          <span
            className="makeStyles-title-296 makeStyles-title-344"
          >
            New Template
          </span>
        </span>
      </button>
      <div
        className="MuiCircularProgress-root makeStyles-buttonProgress-299 makeStyles-buttonProgress-346 MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
        role="progressbar"
        style={
          Object {
            "height": 24,
            "width": 24,
          }
        }
      >
        <svg
          className="MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
            cx={44}
            cy={44}
            fill="none"
            r={19.5}
            strokeWidth={5}
            style={Object {}}
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
